src/ioctls/vm.rs

```diff
@@ -1106,6 +1106,32 @@ impl AsRawFd for VmFd {
     }
 }
 
+/// Creates a GIC device for Arm architectures using either the KVM_CREATE_DEVICE_TEST flags or not.
```

Comment:
I think this should only specify that it creates a dummy GIC device (dummy because instead of a valid fd, we just write 0).
Isn't arm already implied when we say GIC? We should remove the part with "using either the KVM_CREATE_DEVICE_TEST flags or not" as this is covered by the Arguments section.

Reply:
Ah yes sorry for being wordy...... Thanks very much for the suggstion, I have changed it and will push again after our discussion about this `nr_irq` stuff.

Reply:
Don't worry about it, I am not bothered at all. I'll reply to the other comments in line.